Examples/ArticleExample/ArticleExampleViewController.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  MediumViewController.swift
+//  CollectionKit
+//
+//  Created by yansong li on 2017-09-03.
+//  Copyright © 2017 lkzhao. All rights reserved.
+//
+
+import CollectionKit
+import UIKit
+
+class ArticleExampleViewController: UIViewController {
+  
+  var collectionView: CollectionView = {
+    let collectionView = CollectionView()
+    collectionView.backgroundColor = .white
+    return collectionView
+  }()
+
+  override func loadView() {
+    self.view = collectionView
+  }
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    let provider = CollectionProvider(
+      data: articles(count: 8),
+      viewUpdater: { (view: ArticleView, data: ArticleData, at: Int) in
+        view.populate(article: data)
+      },
+      layout: FlowLayout(insets: UIEdgeInsets(top: 20, left: 16, bottom: 16, right: 20), padding: 30),
```

Comment:
Sorry, this inset that I wrote is incorrect.

Reply:
top and bottom should be 20, left right should be 16

Reply:
Done